src/lang/KclSingleton.ts

```diff
@@ -501,11 +494,6 @@ export class KclManager {
     }
 
     if (execute) {
-      // Call execute on the set ast.
-      // Update the code state and editor.
-      codeManager.updateCodeEditor(newCode)
-      // Write the file to disk.
-      await codeManager.writeToFile()
```

Comment:
Nice, so this is the beating heart of the PR

Reply:
Yeah. It's crazy how far the repercussions of these 2 lines goes.